src/util.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Get the current totals.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getCurrentTotal( cart ) {
+	if ( undefined !== cart ) {
+		const total_price = cart.cartTotals.total_price;
+		const minor_unit = cart.cartTotals.currency_minor_unit;
+
+		return total_price / Math.pow( 10, minor_unit );
+	}
+	return 15;
+}
+
+/**
+ * Get the minor unit.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getMinorUnit( cart ) {
+	if ( undefined !== cart ) {
+		return cart.cartTotals.currency_minor_unit;
+	}
+	return '';
```

Comment:
Why do we return a string? The function says number, so perhaps return `0`?

Reply:
Good question why I'm return a string. Does not make any sense here, and I changed it already.